GitCommands/CustomDiffMergeToolCache.cs

```diff
@@ -68,15 +68,17 @@ public async Task<IEnumerable<string>> GetToolsAsync(GitModule module, int delay
                 }
             }
             catch
+            {
+                // No action.
```

Comment:
Non-cancellation exceptions are forwarded by the `.FileAndForget()` in `LoadCustomDiffMergeTools()`.
Why should we swallow them?

Reply:
Keep this as non important, similar to git-status refreshes in the background, to avoid user confusion for non critical issues.

Reply:
Unless you request changes, I would like to merge this tomorrow.

Reply:
I don't. We can ask for the Git Command Log in case of issues.